src/main/java/org/jabref/gui/groups/GroupTreeController.java

```diff
@@ -87,7 +90,13 @@ public void initialize() {
                 this::updateSelection
         );
 
-        viewModel.filterTextProperty().bind(searchField.textProperty());
+        // We try to to prevent publishing changes in the searchfield directly to the search task that takes some time
+        // for larger group structures.
+        final Timer searchTask = FxTimer.create(Duration.ofMillis(400), () -> {
+            LOGGER.info("Run Search " + searchField.getText());
```

Comment:
I think debug level is enough.

Reply:
That log is really only for testing so that I could inspect when a new search was triggered and if my "reset the timer on new key" does actually work. This will be removed.

Reply:
Can the log level be changed to "debug" and the logging settings adjusted accordingly for you?